org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/TestUtils.java

```diff
@@ -59,6 +59,8 @@ public class TestUtils {
 	private TestUtils() {
 		// this class shouldn't be instantiated
 	}
+	
+	private static List<File> tempFiles = new ArrayList<>();
 
```

Comment:
This adds a second static field with the same name `tempFiles` to the class.

![image](https://user-images.githubusercontent.com/426959/178364468-008ed067-7de6-40c1-9a97-d4dfd48bc1eb.png)

Reply:
Sorry, was a merge clash caused by this part of the original PR being cherry picked out and integrated independently a few weeks ago. Fixed anyway!